wsf_scraping/spiders/who_iris_spider.py

```diff
@@ -53,9 +52,6 @@ def start_requests(self):
         """ This sets up the urls to scrape for each years.
         """
 
-        # Check that the database is set up with the correct columns
-        check_db()
```

Comment:
Where is that check done now? Or is it not needed?

Reply:
Every database check is now centralised into the pipelines.py file